app/src/ui/rebase/rebase-flow.tsx

```diff
@@ -323,55 +112,33 @@ export class RebaseFlow extends React.Component<
     targetBranch: string,
     commits: ReadonlyArray<CommitOneLine>
   ) => {
-    if (this.state.step.kind !== RebaseStep.ChooseBranch) {
-      throw new Error(`Invalid step to start rebase: ${this.state.step.kind}`)
+    if (this.props.step.kind !== RebaseStep.ChooseBranch) {
+      throw new Error(`Invalid step to start rebase: ${this.props.step.kind}`)
```

Comment:
Is this an error that would be displayed in the UI? If so, how would the user get out of an invalid step? Also, I'm assuming this is necessary because the user may initialize a rebase from the CLI.

Reply:
To save duplicate comments, I'll point to the other discussion that overlaps with this https://github.com/desktop/desktop/pull/7203#discussion_r272150557